### PR TITLE
Improve JUPDialog Styling

### PR DESCRIPTION
**Pull request by outragedhuman**
Setting fullWidth and maxWidth provide a nice consistent dialog width which then changes to full screen on the `sm` breakpoint.

### CODE REVIEW

File: src/components/JUPTable/JUPTable.tsx [hunk 1]
```diff
@@ -33,17 +33,23 @@ export interface IHeadCellProps {
...
 const TableTitle: React.FC<ITableTitleProps> = ({ title, path, DisplayedComponents }) => {
   return (
     <>
-      <TitleText variant="h6" id="tableTitle">
-        <SLink href={path}>{title}</SLink>
-        {DisplayedComponents}
-      </TitleText>
+      {path ? (
+        <TitleText variant="h6" id="tableTitle">
+          <SLink href={path}>{title}</SLink>
```
Comment (vwalsh) [inline src/components/JUPTable/JUPTable.tsx:45]: why not just wrap the SLINK with the IF statement here? why the double titletext ?

File: src/components/SearchBar/components/AddressBook/AddressBook.tsx [hunk 2]
```diff
@@ -1,26 +1,13 @@
...
-} from "@mui/material";
+import { Button, DialogContent, FormGroup, IconButton, Input, Stack, styled } from "@mui/material";
 import ImportContactsIcon from "@mui/icons-material/ImportContacts";
 import JUPDialog from "components/JUPDialog";
 import { useSnackbar } from "notistack";
 import { messageText } from "utils/common/messages";
 import { isValidAddress } from "utils/validation";
 import useBreakpoint from "hooks/useBreakpoint";
+import JUPTable from "components/JUPTable";
+import { IHeadCellProps, ITableRow } from "components/JUPTable/JUPTable";
```
Comment (vwalsh) [inline src/components/SearchBar/components/AddressBook/AddressBook.tsx:10]: consider moving these interfaces into the components/JUPTable location. aka, move them into the index.ts file.

also implies that they would be required to be imported within the component itself, which is easy to do.

**Review by vwalsh — COMMENTED** [review]
see comments

File: src/components/JUPTable/JUPTable.tsx (hunk 1 above)
Comment (outragedhuman) [reply src/components/JUPTable/JUPTable.tsx]: fixed

File: src/components/SearchBar/components/AddressBook/AddressBook.tsx (hunk 2 above)
Comment (outragedhuman) [reply src/components/SearchBar/components/AddressBook/AddressBook.tsx]: fixed

**Comment by outragedhuman** [comment]
reviewed fixed alongside V

**Pull request merged**